core/src/uix/core/alpha.cljc

```diff
@@ -244,7 +244,7 @@
        (into [:> `(.-Provider ~ctx) {:value value}]
              children)
        `(binding [~ctx ~value]
-          ~(into [:<>] children)))))
+          ~(into [:<>! `(fn [f#] (binding [~ctx ~value] (f#)))] children)))))
```

Comment:
why do we need a new Hiccup form here?

Reply:
You need to capture the context and bind it when the hiccup is resolved, I don't love this solution but its pretty simple, and only for SSR. #66 explains the issue.

Reply:
Got it. I'd suggest to come up with a more unique and explicit name, something like `:uix.core.alpha/bind-context`. Also unit tests would be great.